src/main/java/pages/TestTypeDetailsPage.java

```diff
@@ -17,6 +17,28 @@
     private static final String MOST_RECENT_INSTALLATION_CHECK_DONE_BUTTON_ID = "Done";
     private static final String MOST_RECENT_INSTALLATION_CHECK_CANCEL_BUTTON_ID = "Cancel";
     private static final String SEATBELT_TITLE_ID = "Seatbelt installation check";
+    private static final String TEST_RESULT_ID = "Test result Select arrow forward";
+    private static final String SET_LEC_TEST_TO_PASS_ID = "Pass";
+    private static final String SET_LEC_TEST_TO_FAIL_ID = "Fail";
+    private static final String CANCEL_LEC_TEST_ID = "Cancel";
+    private static final String CERTIFICATE_NUMBER_LABEL_ID = "CERTIFICATE NUMBER";
+    private static final String LP_LABEL_ID = "LP";
+    private static final String CERTIFICATE_NUMBER_INPUT_FIELD_XPATH = "//XCUIElementTypeOther[@name=\"Ionic App\"]/XCUIElementTypeOther[7]/XCUIElementTypeTextField";
```

Comment:
please try to identify by class name XCUIElementTypeTextField

Reply:
done